bitnami/contour/Chart.yaml

```diff
@@ -22,4 +22,4 @@ maintainers:
 name: contour
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/contour
-version: 12.0.1
+version: 12.2.0
```

Comment:
Please ensure to update the chart version in accordance with the semver principles outlined in our [contributing guidelines](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#technical-requirements).
The current version is [12.0.4](https://github.com/bitnami/charts/blob/main/bitnami/contour/Chart.yaml#LL25C3-L25C16), so it should be incremented to 12.1.0.

Reply:
done!